ts/IterableInboxMessageDisplay.tsx

```diff
@@ -70,17 +71,27 @@ const IterableInboxMessageDisplay = ({
    })
 
    const handleHTMLMessage = (event: any) => {
-      if(event.nativeEvent.data === 'iterable://delete') {
+      let url = event.nativeEvent.data
+      if(url === 'iterable://delete') {
          deleteRow(rowViewModel.inAppMessage.messageId)
          returnToInbox()
-      } else if(event.nativeEvent.data === 'iterable://dismiss') {
+      } else if(url === 'iterable://dismiss') {
          returnToInbox()
       } else {
-         if(Iterable.savedConfig.urlHandler) {
-            Iterable.savedConfig.urlHandler(event.nativeEvent.data, context)
+         if(Iterable.savedConfig.urlHandler && Iterable.savedConfig.urlHandler(url, context)) {
+            Iterable.savedConfig.urlHandler(url, context)
+            returnToInbox()
          }
+      }
+   }
+
+   const openExternalURL = (event: any) => {
```

Comment:
Also not needed for this PR, but I think we should try to use `function` more, I think the advantages are helpful for a method like this, like in [this article](https://softwareengineering.stackexchange.com/questions/364086/why-use-const-foo-instead-of-function-foo).

Reply:
My vote goes for function too 👍

Reply:
I will use function going forward.